image_tools/src/options.cpp

```diff
@@ -76,6 +77,9 @@ bool parse_command_options(
     if (burger_mode != nullptr) {
       ss << " -b: produce images of burgers rather than connecting to a camera" << std::endl;
     }
+    if (topic != nullptr) {
+      ss << " -t TOPIC: use topic TOPIC instead of the default \"image\"" << std::endl;
```

Comment:
I agree duplicating it here isn't ideal, and with the added print at publisher creation time we don't need it anymore.  To be honest, this entire option parsing function could be done better; I think it would be nicer to use/write some sort of library that is more similar to Python's argparse.  In any case, I'll change this.